app/javascript/react/screens/App/Overview/OverviewReducer.js

```diff
@@ -100,7 +100,7 @@ export const initialState = Immutable({
   isFetchingNetworks: false,
   isRejectedNetworks: false,
   scheduleMigrationModal: false,
-  scheduleMigrationPlanId: null,
+  scheduleMigrationPlan: {},
```

Comment:
I think you want to set the default as an empty string here due to [this line](https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/514/files#diff-a64dbfef583d0adfd0466639743beb9cR194)

Reply:
eeeeek nope, good catch, that needs to be an object, cuz the plan is an object